src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/NoneAttestationEnclaveProvider.cs

```diff
@@ -3,21 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
-using System.Runtime.Caching;
 using System.Security.Cryptography;
-using System.Text;
 using System.Threading;
-using System.Threading.Tasks;
-using System.Collections.Concurrent;
 
 namespace Microsoft.Data.SqlClient
 {
-    internal class SimulatorEnclaveProvider : EnclaveProviderBase
+    internal class NoneAttestationEnclaveProvider : EnclaveProviderBase
     {
         private static readonly int EnclaveSessionHandleSize = 8;
+        private const int DiffieHellmanKeySize = 384;
+        private const int NoneAttestationProtocolId = 2;
```

Comment:
This value equals to `SqlConnectionAttestationProtocol.SIM` which builds only with `ENCLAVE_SIMULATOR=true` at build time. Could you add some notes to explain the source of the value **2**?